docs/recipes/MaestroSetup.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Maestro Setup
+description: Setting up e2e testing with Maestro in Ignite
+tags:
+  - Maestro
+  - testing
+last_update:
+  author: Dan Edwards
+publish_date: 2023-01-06
+---
+
+# Setting Up Maestro in Ignite
+
+## Overview
+
+E2E testing is a critical part of any application but it can be difficult to set up and maintain. [Maestro](https://maestro.mobile.dev/) is a tool that promises to be easy to set up and maintain e2e tests. This recipe will walk you through setting up Maestro in your Ignite project.
+
+## Maestro Installation
+
+We're going to start by installing Maestro via the terminal. To do this, we'll need to run the following command:
+
+```bash
+    curl -Ls "https://get.maestro.mobile.dev" | bash
+```
+
+Once the installation is complete, you'll be ready to create your first Maestro flow!
+
+## Creating our first Maestro Flow
+
+To start out, we're going to create a folder to hold our Maestro flows. Let's do this by adding a folder in the root of our Ignite project called `.maestro`. Once that's done we can create our first flow in a file called `Login.yaml`
+
+With this flow we want to open up our app and then login with the default credentials. We can do this by adding the following to our `Login.yaml` file:
+
+```yaml
+#flow: Login
+#intent:
+# Open up our app and use the default credentials to login
+# and navigate to the demo screen
+
+appId: com.maestroapp # the app id of the app we want to test, in my case it's the com.maestroapp
```

Comment:
Instead of "in my case" let's provide guidance on where to look this up. What file it lives in and/or reference the part of the ignite new command where they would have submitted this value